P/pg_tmp/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "pg_tmp"
```

Comment:
Why this name?

Reply:
The executable that should be built is supposed to be `pg_tmp`. I hit the wrong button by mistake and launched the PR instead of auditing it. I am investigating now why it didn't build correctly

Reply:
Note that we use as name the name of the upstream package, not that of a product.  What about `ephemeralpg`?  Note that when you change the value of this variable you need to rename the directory where this file is accordingly.

Reply:
That's much better. Thanks!

Apparently the other executables are fine, so this should be good to go.